homeassistant/components/mqtt/strings.json

```diff
@@ -19,10 +19,10 @@
           "username": "[%key:common::config_flow::data::username%]",
           "password": "[%key:common::config_flow::data::password%]",
           "advanced_options": "Advanced options",
-          "certificate": "Path to custom CA certificate file",
+          "certificate": "Upload custom CA certificate file",
           "client_id": "Client ID (leave empty to randomly generated one)",
-          "client_cert": "Path to a client certificate file",
-          "client_key": "Path to a private key file",
+          "client_cert": "Upload client certificate file",
```

Comment:
We should use references to the other values so we don't have translators have to do the work twice.

See example in `homekit`